tests/B2CWebAppCallsWebApi/Client/appsettings.json

```diff
@@ -4,10 +4,10 @@
         "ClientId": "fdb91ff5-5ce6-41f3-bdbd-8267c817015d",
         "Domain": "fabrikamb2c.onmicrosoft.com",
         "SignedOutCallbackPath": "/signout/B2C_1_susi",
-        "SignUpSignInPolicyId": "b2c_1_susi",
-        "ResetPasswordPolicyId": "b2c_1_reset",
+        "SignUpSignInPolicyId": "b2c_1_susi_v3",
+        "ResetPasswordPolicyId": "b2c_1_reset_v3",
         "EditProfilePolicyId": "b2c_1_edit_profile", // Optional profile editing policy
-        "ClientSecret": "[Copy the client secret added to the app from the Azure portal]"
+        "ClientSecret": "secret-here"
         //"CallbackPath": "/signin/B2C_1_sign_up_in"  // defaults to /signin-oidc
```

Comment:
Do we need to comment the callback path out ?

Reply:
i didn't change that

Reply:
I got confused by the syntax highlighting :)